content/blog/2019-09-25-c-implementation.md

```diff
@@ -0,0 +1,189 @@
++++
+title = "C Implementation of Bril"
+extra.author = "Bhargava S. Manja"
+extra.bio = """
+  [Bhargava S. Manja](https://github.com/bhargee) is a first year PhD student
+interested in computer vision, fast computing, and domain specific languages
+"""
++++
+
+## Project Report 1: C Implementation of Bril
```

Comment:
No need for another copy of the title; the template will display your title for you.